RoadMap_SwiftUI/RoadMap_SwiftUI/ContentView.swift

```diff
@@ -8,14 +8,97 @@
 import SwiftUI
 
 struct ContentView: View {
+    private enum Constants {
+        static let changedText = "Text changed"
+        static let showButtonText = "Показать"
+        static let firstAlertLabelText = "Пример Alert"
+        static let secondAlertLabelText = "Пример Alert c 2 кнопками и логикой"
+        static let firstActionSheetText = "Пример ActionSheet"
+        static let secondActionSheetText = "Пример ActionSheet c кнопками и логикой"
+        static let revertText = "Вернуть обратно"
+        static let questionText = "Изменить текст?"
+        static let yesText = "Да"
+        static let areYouSureText = "Вы уверены?"
+    }
+    
     var body: some View {
-        VStack {
-            Image(systemName: "globe")
-                .imageScale(.large)
-                .foregroundColor(.accentColor)
-            Text("Hello, RoadMap!")
+        VStack(alignment: .center,
+               spacing: 20) {
+            Text(Constants.firstAlertLabelText)
+            showFirstAlert()
+            if isTapped { Text (Constants.changedText) } else {
+                Text(Constants.secondAlertLabelText)
+            }
+            showSecondAlert()
+            Text(Constants.firstActionSheetText)
+            showActionSheet()
+            Text(Constants.secondAlertLabelText)
+            showSecondActionSheet()
+        }
+    }
+    
+    @State private var isFirstButtonTapped = false
+    @State private var isError = false
+    @State private var isTapped = false
+    @State private var isActionSheetButtonTapped = false
+    @State private var isSecondActionSheetTapped = false
+    
+    private func showFirstAlert() -> some View {
```

Comment:
```suggestion
    private var showFirstAlert: some View {
```
если нет параметров, то переделываем на вычисляемое свойство